filter-effects/Overview.bs

```diff
@@ -370,6 +373,11 @@ Note: For some filter functions the default value for omitted values differes fr
 
         Note: Standard deviation is different to 'box-shadow' s blur radius.
 
+        Note: A true Gaussian blur has theoretically infinite extent, but the
```

Comment:
* Why is this non-normative?

* Is the term "three-pass box blur approximation" defined anywhere we can link to?

Shouldn't this say  instead, in non-normative text, that "the ink overflow rectangle is for a gaussian blur is the extent of the drawing used in the implementation of that blur by an implementation", and in a non-normative text say "theoretically a gaussian blur has infinite extent, but that is not implementable so all implementations use some approximation, of which three-pass blur is a common one. In that situation, the ink overflow rectangle will be the output image are of the three-pass blur"

Reply:
Reworded your suggestion a bit, PTAL.